docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -16,22 +16,22 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 #### PHONE_CALL
 
 !!! MUST
-1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
+1. Change audioStreamingState to NOT_AUDIBLE for all apps (keep hmiLevel) when active call on HMI has been started.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Send notification with appropriate parameter value when the event ends.
```

Comment:
I would just remove point 2 in the original, no need to rewrite most of this section
```
1. Send notification with appropriate parameter value when active call on HMI has been started/ended.

2. Resume the applications to their original audio streaming state prior to the phone call event in the HMI when the event ends (see note below).
```

Reply:
We think there is no problem to modify according to your suggestion.